gotests_test.go

```diff
@@ -66,10 +66,13 @@ func TestGenerateTests(t *testing.T) {
 		}, {
 			name: "Target test file",
 			args: args{
-				srcPath: `testdata/test100_test.go`,
+				srcPath:  `testdata/test103_test.go`,
+				only:     regexp.MustCompile("wrapToString"),
```

Comment:
If you don't specify the "only" parameter, does it generate tests for the test functions? IRC the reason for not including "_test" files was to avoid generating tests for test functions like `TestBarBar100` becoming `TestTestBarBar100`. If so, please add a condition not to generate tests for test functions in a test file, and another unit test to verify the behavior. Otherwise this pull request looks good to me.

Reply:
Ok. I will fix it.

Reply:
Done.